ctapipe/containers.py

```diff
@@ -432,7 +407,30 @@ class MCEventContainer(Container):
         "2(mu-), 100*A+Z for nucleons and nuclei,"
         "negative for antimatter.",
     )
-    tel = Field(Map(MCCameraEventContainer), "map of tel_id to MCCameraEventContainer")
+
+
+class SimulatedCameraContainer(Container):
+    """Contains all fields of the DL1CameraContainer, but adds fields for simulated
```

Comment:
Not really true, since we do not have peak times here. (We could calculate them in the simtel event source, see #1341

Reply:
Only problem is it needs to be clear that there is an unknown offset between these simulated mean times, and the times extracted from the waveform

Reply:
Yes, these times are in the CORSIKA system and I don't know anything in the SimTel output that relates CORSIKA times to the trigger time

Reply:
I'll write Konrad

Reply:
That was an old comment from when this originally inherited from DL1CameraContainer -  I'll just update it to be more accurate. Not sure if we really need the simulated mean times or not. Could be useful for benchmarking, I guess.

Reply:
Konrad answered, and it is not easy to get these CORSIKA times converted to times relative to the array trigger, so I would leave it out for someone who really needs this to implement.